crates/collab/src/rpc.rs

```diff
@@ -992,13 +992,7 @@ impl Server {
             tracing::Span::current().record("connection_id", format!("{}", connection_id));
             tracing::info!("connection opened");
 
-            let http_client = match IsahcHttpClient::new() {
-                Ok(http_client) => Arc::new(http_client),
-                Err(error) => {
-                    tracing::error!(?error, "failed to create HTTP client");
-                    return;
-                }
-            };
+            let http_client = IsahcHttpClient::new(None);
```

Comment:
We should probably roll back this change. We probably don't want to remove isahc from collab, even if we do switch to `ureq`